manual/vmc.tex

```diff
@@ -28,37 +28,87 @@ \section{Variational Monte Carlo}
 
 Additional information:
 \begin{itemize}
-\item \texttt{walkers}. The initial default number of walkers is 1 but in the CPU branch this number will be overwritten as the number of OpenMP threads if the user requested number is smaller than the number of threads.
+\item \texttt{walkers}. The initial default number of walkers is 1 per OpenMP thread or per MPI task if threading is disabled.
```

Comment:
The overwriting behaviour needs to be kept here.

Reply:
Updated